tests/server/test_daemon.py

```diff
@@ -243,13 +238,6 @@ async def test_broadcast_transaction(daemon):
     assert await daemon.broadcast_transaction(raw_tx) == tx_hash
 
 
-@pytest.mark.asyncio
-async def test_relayfee(daemon):
-    response = {"relayfee": sats, "other:": "cruft"}
-    daemon.session = ClientSessionGood(('getnetworkinfo', [], response))
-    assert await daemon.getnetworkinfo() == response
```

Comment:
I see this was dead code as the next test redefines the name.
-- but also, `sats` is not even defined. maybe we could add flake8 or similar to the test suites

Reply:
Agreed. Will be easy, except for the first big list of issues it will find :)

Reply:
FWIW, vanilla flake8 says:
```
2     E111 indentation is not a multiple of four
1     E114 indentation is not a multiple of four (comment)
1     E116 unexpected indentation (comment)
2     E117 over-indented
6     E127 continuation line over-indented for visual indent
9     E201 whitespace after '('
8     E202 whitespace before ')'
3     E203 whitespace before ':'
1     E211 whitespace before '('
1     E221 multiple spaces before operator
1     E225 missing whitespace around operator
1     E231 missing whitespace after ','
2     E265 block comment should start with '# '
38    E302 expected 2 blank lines, found 1
1     E303 too many blank lines (3)
2     E305 expected 2 blank lines after class or function definition, found 1
2     E306 expected 1 blank line before a nested definition, found 0
4     E402 module level import not at top of file
317   E501 line too long (85 > 79 characters)
2     E712 comparison to True should be 'if cond is True:' or 'if cond:'
1     E731 do not assign a lambda expression, use a def
18    F401 'electrumx.server.controller.Controller' imported but unused
21    F541 f-string is missing placeholders
1     F811 redefinition of unused 'test_relayfee' from line 246
2     F821 undefined name 'utxos'
15    F841 local variable 'strDZeel' is assigned to but never used
2     W293 blank line contains whitespace
1     W391 blank line at end of file
```